doozer/doozerlib/cli/images_streams.py

```diff
@@ -1123,7 +1123,7 @@ def check_if_upstream_image_exists(upstream_image):
 
             yellow_print(f'Upstream dockerfile does not match desired state in {public_repo_url}/blob/{public_branch}/{dockerfile_name}')
 
-            first_commit_line = f"Updating {image_meta.get_component_name()} image to be consistent with ART"
+            first_commit_line = f"Updating {image_meta.get_component_name()} image to be consistent with ART for {runtime.group_config.name}"
```

Comment:
Use `group_config.vars.MAJOR` and `group_config.vars.MINOR`.

Reply:
updated，version already defined in line 863/4